SPECS/ca-certificates-prebuilt/ca-certificates-prebuilt.spec

```diff
@@ -0,0 +1,42 @@
+Summary:        Prebuilt version of ca-certificates package
+Name:           ca-certificates-prebuilt
+
+Version:        20200720
+Release:        1%{?dist}
+License:        MIT
+
+URL:            https://hg.mozilla.org
+Group:          System Environment/Security
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+
+BuildArch:      noarch
+
+BuildRequires:  ca-certificates-base
+Conflicts:      ca-certificates-base
+Conflicts:      ca-certificates
```

Comment:
It also conflicts with `ca-certificates-microsoft` and potentially with `ca-certificates-shared`.